tests/getEntities.js

```diff
@@ -0,0 +1,227 @@
+const expect = require('chai').expect;
+const parse = require('..').convert;
+const b_ = require('@bem/entity-name').create;
+
+it('should return an array', () => {
+    expect(parse({ block: 'button2' })).to.be.an('Array');
+});
+
+it('should return array of zero length if bemjson is empty', () => {
+    expect(parse({})).to.have.lengthOf(0);
+    expect(parse([])).to.have.lengthOf(0);
+});
+
+describe('immutable', () => {
+    // TODO: Any ideas?
+});
+
+describe('block', () => {
+
+    it('should extract block', () => {
+        expect(parse({ block: 'button2' })).to.eql([b_({ block : 'button2' })]);
+    });
+
+    it('should extract block with simple modifier', () => {
+        expect(parse({ block: 'popup', mods: { autoclosable: true } })).to.eql([
+            b_({ block: 'popup' }),
+            b_({ block: 'popup', mod: { name: 'autoclosable' } })
+        ]);
+    });
+
+    it('should extract block with modifier', () => {
+        expect(parse({ block: 'popup', mods: { autoclosable: 'yes' } })).to.eql([
+            b_({ block : 'popup' }),
+            b_({ block : 'popup', mod : { name : 'autoclosable', val : 'yes' } }),
+            b_({ block : 'popup', mod : { name : 'autoclosable' } })
```

Comment:
I think it is better to change it in another pr https://github.com/bem-sdk/bemjson-to-decl/issues/21